### PR TITLE
Api extractor styling

### PR DESCRIPTION
**Pull request by kkjeer**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Enable the api-extractor task for the styling package.
Remaining work: add the api-extractor task to the CI build (will be addressed in a future PR).

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/styling/etc/styling.api.ts [hunk 1]
```diff
@@ -0,0 +1,647 @@
...
+export function createTheme(theme: IPartialTheme, depComments?: boolean): ITheme;
+
+// @public
+export function focusClear(): IRawStyle;
+
+// @public
+export function fontFace(font: IFontFace): void;
+
+// WARNING: Export "mini" is missing the "const" modifier. Currently the "namespace" block only supports constant variables.
+// WARNING: Export "xSmall" is missing the "const" modifier. Currently the "namespace" block only supports constant variables.
```
Comment (cliffkoh) [inline packages/styling/etc/styling.api.ts]: Are these warnings actionable?

Comment (kkjeer) [reply packages/styling/etc/styling.api.ts]: Looking into it.

Comment (KevinTCoughlin) [reply packages/styling/etc/styling.api.ts]: 🤔 aren't they marked as `const` tho? https://github.com/OfficeDev/office-ui-fabric-react/blob/6e82e75446a80acba84cc29a45d6f705c67fb828/packages/styling/src/styles/fonts.ts#L72

Interested to know how to resolve these warnings.

**Comment by dzearing** [comment]
ERROR: Change file does not contain
@uifabric/charting,@uifabric/dashboard,@uifabric/experiments,@uifabric/fabric-website-resources,@uifabric/foundation,@uifabric/merge-styles,@uifabric/utilities,office-ui-fabric-react.

**Comment by dzearing** [comment]
resetting ci.

**Pull request merged**